src/components/UserInfo/UserInfo.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { shape } from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  StyleSheet, Text, View,
+} from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
+
+import {
+  getPatientName,
+  getDataRange,
+} from '../../resources/fhirReader';
+import { patientSelector } from '../../redux/selectors';
+import { clearPatientData } from '../../features/patient/patientDataSlice';
+
+const UserInfo = ({
+  patientResource, resources,
+}) => {
+  const name = getPatientName(patientResource);
+  const [dataStart, dataEnd] = getDataRange(resources);
+
+  return (
+    <View>
+      <View style={styles.userContainer}>
+        <MaterialIcons name="face" size={40} color="black" style={styles.userIcon} />
```

Comment:
Can you use `<Ionicons name="person-circle-outline" size={24} color="black" />` as it uses the same icon library as the Tab buttons? Perhaps use your size (40) or larger to match to mockup proportions.

Reply:
Done